packages/teuchos/parameterlist/src/Teuchos_ParameterList.hpp

```diff
@@ -811,6 +811,21 @@ TEUCHOSPARAMETERLIST_LIB_DLL_EXPORT bool haveSameValues( const ParameterList& li
     bool verbose = false);
 
 
+/** \brief Returns true if two parameter lists have the same values independent of ordering.
+ *
+ * Two parameter lists may have the same values but may not be identical.  For
```

Comment:
Does this check also include the documentation string and the validator?  It is not clear from this sentence.

Reply:
The next sentence (lines 816-818) explains that the documentation string and validator are ignored.  I basically just copied the documentation from `haveSameValues` and changed "respects ordering" to "not respect ordering".